languages/fsharp/exercises/concept/strings/.docs/after.md

```diff
@@ -0,0 +1,19 @@
+# After
+
+The key thing to remember about F# strings is that they are immutable objects representing text as a sequence of Unicode characters (letters, digits, punctuation, etc.).
+
+Manipulating a string can be done by calling one of its [methods][methods] or [properties][properties], or using one of the functions in the [`String` module][string-module]. As string values can never change after having been defined, all string manipulation methods/functions will return a new string.
+
+A string is delimited by double quote (`"`) characters. Some special characters need [escaping][escaping] using the backslash (`\`) character. Strings can also be prefixed with the at (`@`) symbol, which makes it a [verbatim string][verbatim] that will ignore any escaped characters.
+
+Alternatively, strings can be defined using triple quotes (`"""`), which allows using a double quote in the string without escaping it. [Triple-quoted strings][triple-quoted] can also span multiple lines.
+
+Finally, concatenating strings can be done through the [`+` operator][plus-operator]. For any string formatting more complex than simple concatenation, the `sprintf` function is preferred.
```

Comment:
Do we want to reference `StringBuilder` here as the ideal way to concatenate multiple strings for performance reasons?

Reply:
I'd prefer to not mention it yet, as `StringBuilder` means knowing about state and it is mostly used for performance reasons, which I'd prefer not to bother the student with in this introductory exercise. I've added a note to the `design.md` file that mentions not wanting to teach the `StringBuilder` class.

Reply:
Makes sense to me!